.gitmodules

```diff
@@ -82,3 +82,7 @@
 	path = assets/img/2017
 	url = https://github.com/emergencyindex/projects-2017
 	branch = images
+[submodule "_indexes/2018"]
```

Comment:
ohey, so i don't think you want to include any changes to the .gitmodules file in this PR

Reply:
right right right right right